e2e-tests/test/debug-apis.test.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from "chai";
+import { Wallet } from "zksync-web3";
+import * as hre from "hardhat";
+import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
+import { RichAccounts } from "../helpers/constants";
+import { deployContract, expectThrowsAsync, getTestProvider } from "../helpers/utils";
+
+const provider = getTestProvider();
+
+describe("debug namespace", function () {
+  it("Should return error if block is not 'latest' or unspecified", async function () {
+    expectThrowsAsync(async () => {
+      await provider.send("debug_traceCall", [{ to: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266" }, "earliest"]);
+    }, "block parameter should be 'latest' or unspecified");
+
+    expectThrowsAsync(async () => {
```

Comment:
nit **unrelated to this PR**: `expectThrowsAsync` helper is idiomatically written via `chai-as-promised` https://www.chaijs.com/plugins/chai-as-promised/
We could have a follow up PR to use the idiomatic way instead and reduce helper/util code.


```typescript
return expect(provider.send("debug_traceCall", [{ to: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266" }, "1"])).to.eventually.be.rejectedWith(Error, "block parameter should be 'latest' or unspecified");
```

Reply:
This would still need to be modified as this is a partial string matching for the error message due to the formatting of responses from JSON RPC APIs